### PR TITLE
Add a few new keywords and selectors, fix some of the test suite.

### PR DESCRIPTION
**Pull request by pygy**
This adds support for: 
- `cursor: grab`, `cursor: grabbing`, `width: contains-float`
- `::backdrop`, `:fullscreen` and `::placeholder`.

The tests have improved, but they are still broken...

### CODE REVIEW

File: prefixfree.js [hunk 1]
```diff
@@ -447,10 +449,24 @@ for (var keyword in keywords) {
...
+	'::placeholder': '::placeholder',
+	':input-placeholder': '::placeholder',
+	'::input-placeholder': '::placeholder',
+
+	':read-only': ':read-only',
+	':read-write': ':read-write',
+
+	'::selection': '::selection'
+
+
```
Comment (LeaVerou) [inline prefixfree.js]: I like the idea of a map here, but there should be a default that means "same as key". I don't like this duplication. How about `null` being that value?

**Comment by LeaVerou** [comment]
Thanks! See comment. Yeah, the testsuite needs work… :/

**Comment by pygy** [comment]
Done, and rebased (edit: and rebased again, I had left extra blank lines).

My intuitions about the compressed output were wrong, using `null` and an additional branch makes the .min.gzipped version slightly smaller. I had to compress by hand using https://closure-compiler.appspot.com/home since the `/compile` endpoint used by the `minify` script is silent right now.

**Comment by LeaVerou** [comment]
Thanks, merged!

Regardless of size, DRY code is easier to maintain and in this case, easier to read. Now I can tell at a glance which selectors need substitution instead of just prefixing.

**Comment by pygy** [comment]
Agreed, it's a matter of priorities.

For libs like this I tend to favor speed and size down the line (which doubles as speed) over readability and compensate the increased maintenance overhead with tests, but I can totally understand where you're coming from, especially for code that you don't visit often.

**Comment by coliff** [comment]
in case you hadn't noticed the minified version is currently broken. (no javascript code is in the file).

**Comment by pygy** [comment]
Oooh my bad, that happened because the closure compiler was not responding, I thought I had left it out... I must have `git add -u` at some point...

**Comment by coliff** [comment]
Cool. Are you updating the version number too? It's been on v1.0.7 for a while - even though there have been a number of changes and improvements. It'd be helpful if the version number was bumped.

**Comment by pygy** [comment]
The bower file is at v1.0.10...

I don't know if the additional selectors/keywords count as bug fixes (1.0.11) or as new features (1.1.0)... I don't know if the changed behavior of `fixSelector` has to be taken into account (since it is documented as mostly private)... in that case, the is backwards incompatible we'd be up to v2...

**Comment by LeaVerou** [comment]
No need for v2, this is an undocumented method that's not expected to be used much outside -prefix-free.

**Pull request merged**